cfy_manager/components/postgresql_client/postgresql_client.py

```diff
@@ -75,8 +73,12 @@ def _install(self):
 
         files.copy_notice(POSTGRESQL_CLIENT)
 
-        self._create_postgres_group()
-        self._create_postgres_user()
+        db_server_username = \
+            config[POSTGRESQL_CLIENT].get('server_username', None)
```

Comment:
Wouldn't it make more sense to add this in to the config.yaml, perhaps as 'server_superuser_name' to make it clear the level of permissions we expect it to have?

Reply:
This change includes lowering our requirement of permissions to not mandate superuser level, as we do not always have it (Azure DBaaS for one...)

Reply:
Right, but still it should be in the config.yaml and therefore not require a .get here, given the way we generate the config based on a merge of defaults and what the user provides.